via/resources.py

```diff
@@ -0,0 +1,27 @@
+"""Context objects for views."""
+from pyramid.httpexceptions import HTTPBadRequest
+
+# pylint: disable=too-few-public-methods
+
+
+class URLRoute:
```

Comment:
```suggestion
class URLResource:
```

These are [called "resources" in Pyramid](https://docs.pylonsproject.org/projects/pyramid/en/latest/narr/resources.html) so I think `FooResource` is a better naming style for them. "Routes" are something else entirely

Reply:
Yep.